bin/u-stool

```diff
@@ -41,7 +41,8 @@ sdata=(
   [maas-eph-daily]="$MAAS_v2/daily/streams/v1/com.ubuntu.maas:daily:v2:download.sjson"
 
   [maas3-daily]="$MAAS_v3/daily/streams/v1/index.sjson"
-  [maas3-eph-daily]="$MAAS_v3/daily/streams/v1/com.ubuntu.maas:daily:v3:download.sjson"
+  [maas3-eph-daily]="$MAAS_v3/daily/streams/v1/com.ubuntu.maas:stable:v3:download.sjson"
```

Comment:
```suggestion
  [maas3-eph-daily]="$MAAS_v3/candidate/streams/v1/com.ubuntu.maas:candidate:v3:download.sjson"
```

In the beginning there was daily and release, now there is candidate and stable.

Images land in the candidate stream first, then get promoted to the stable stream after testing (and after the e.g. Ubuntu release they contain are themselves stable)

Reply:
updated.
thanks.